### PR TITLE
use pkgcompiler for julia

### PR DESCRIPTION
**Pull request by Moelf**

### CODE REVIEW

**Comment by niklas-heer** [comment]
@Moelf thanks for your contribution, but the CI doesn't seem to work.

Also, I don't think you would normally execute your Julia scripts that way.

File: Earthfile [hunk 1]
```diff
@@ -114,12 +114,14 @@ java:
...
   # We have to use a special image since there is no Julia package on alpine 🤷‍♂️
-  FROM julia:1.6.7-alpine3.16
+  FROM julia:1.8.2-alpine3.16
   COPY ./src/rounds.txt ./
   COPY +build/scbench ./
 
   COPY ./src/leibniz.jl ./
-  RUN --no-cache ./scbench "julia leibniz.jl" -i $iterations -l "julia --version" --export json --lang "Julia"
+  RUN julia -e 'using Pkg; Pkg.add("PackageCompiler"); using PackageCompiler; cre
+ate_sysimage(; sysimage_path="mainjl.so",  precompile_execution_file="leibniz.jl")'
```
Comment (darsnack) [inline Earthfile:123]: Should fix the CI hopefully

```suggestion
  RUN julia -e 'using Pkg; Pkg.add("PackageCompiler"); using PackageCompiler; create_sysimage(; sysimage_path="mainjl.so",  precompile_execution_file="leibniz.jl")'
```

Comment (niklas-heer) [reply Earthfile:123]: @darsnack thank you! What will be the output of that command? What I mean does it save the compiled program somewhere?

Comment (Moelf) [reply Earthfile]: In the same direcory, but not xompiled program, just an image to same more compiled stuff, Julia runtime still starts up every single time (which is 98% of the time you're plotting

**Comment by mcabbott** [comment]
Would be interesting to see how big the change is. If more than a little, then perhaps this ought to be one of two separate Julia entries -- much as there are two Python and two Clojure ones.

> I don't think you would normally execute your Julia scripts that way.

Depends who! I would guess most people execute things in an already-running session. Either directly or through some tool like VSCode.

But CI often builds docs etc. by running just `julia make.jl`, a few ms isn't worth any extra complexity. 

This example is too quick for https://github.com/dmolina/DaemonMode.jl to help out. (Which is a command-line way to keep a session alive.)

**Comment by Moelf** [comment]
Looks like we need a g++ installation?

**Comment by niklas-heer** [comment]
combined_results

Okay, it worked. Thank you very much for the contribution @Moelf!

**Comment by niklas-heer** [comment]
Here are the results: https://niklas-heer.github.io/speed-comparison/pages/2022-10-15T223909.html

**Pull request merged**